### ISSUE TITLE
Changing the position of a h3d.scene.Object within the sync method doesn't update its children

### ISSUE THREAD

**Issue by alexpana** [HeapsIO/heaps#1252]
Here is the code for reference: https://github.com/alexpana/path-of-haxe/blob/2da60ec47f7974d444fefdb442e72dbd4f20b248/src/Bullet.hx#L20

The `Bullet` class overrides `sync` to update its own position. Since the implementation of `syncRec` immediately clears the `posChanged` flag of the object, I'm forced to manually sync the position of the children, otherwise they never inherit the position of the bullet (the mesh remains in place).

What would be an idiomatic way of implementing this?

**Comment by ncannasse** [comment]
Yes we noticed that recently and that's not the case in H2D, only H3D so it should be fixed.
Ping @clementlandrin

**Comment by clementlandrin** [comment]
Hello, issue has been fixed with last commit on heaps : 04deb71819f9642d5b47c0838bc6f812b9cbb9ff 
https://github.com/HeapsIO/heaps/commit/04deb71819f9642d5b47c0838bc6f812b9cbb9ff#diff-524bb7c3669a294692e85a7bb851a4cb60f7b8038a4b16c2eef8d0bee84f5917R785
